### PR TITLE
Clean multi-fix

### PR DESCRIPTION
**Pull request by xtrapwr**
So ... this should be all the small fixes, only without the merge conflicts.

### CODE REVIEW

File: stockmaster.js [hunk 1]
```diff
@@ -548,7 +550,7 @@ async function liquidate(ns) {
 /** @param {NS} ns **/
 /** @param {Player} playerStats **/
 async function tryGet4SApi(ns, playerStats, bitnodeMults, budget) {
-    if (ns.stock.has4SDataTIXAPI()) return false; // Only return true if we just bought it
+    if (ns.stock.has4SData()) return false; // Only return true if we just bought it
```
Comment (alainbryden) [inline stockmaster.js:553]: I think this was right how it wss

Comment (xtrapwr) [reply stockmaster.js:553]: I think you are correct as well, let me fix that

Comment (xtrapwr) [reply stockmaster.js]: Swapped that back

File: work-for-factions.js [hunk 2]
```diff
@@ -422,7 +422,7 @@ async function earnFactionInvite(ns, factionName) {
     // Hash for special-case factions (just 'Daedalus' for now) requiring *either* hacking *or* combat
     if (reqHackingOrCombat.includes(factionName) && deficientStats.length > 0 && (
         // Compare roughly how long it will take to train up our hacking stat
-        (requiredHackByFaction[factionName] - player.hacking) / classHeuristic('hacking') <
+        (requiredHackByFaction[factionName] - player.skills.hacking) / classHeuristic('hacking') <
```
Comment (alainbryden) [inline work-for-factions.js:425]: The function `heuristic` 10 lines up needs to be fixed too if you want to tackle it at the same time. Change
```
player[`${stat}_mult`]
```
to
```
player.mults[stat]
```
and
```
player[`${stat}_exp_mult`]
```
to
```
player.stats[`${stat}_exp`]
```

**Review by alainbryden — CHANGES_REQUESTED** [review]

File: work-for-factions.js (hunk 2 above)
Comment (xtrapwr) [reply work-for-factions.js:425]: Change made. That does make sense. Still learning, thanks

**Review by alainbryden — APPROVED** [review]
Doing great. Keep em coming!

**Pull request merged**